main/index.js

```diff
@@ -59,15 +59,19 @@ const checkForUpdates = () => {
     return false;
   }
 
+  const checkForUpdates = async () => {
+    try {
+      await autoUpdater.checkForUpdates();
+    } catch {}
```

Comment:
Should there be a log here or something to catch it in developer or `Console.app`, not sure?

Reply:
Well the updater doesn't run in developer mode. Do you think there's value in logging it to Console.app? Usually these are the errors like connection timed out etc

Reply:
I think this is fine, I test the updater every version manually anyways 👍